src/pages/productListing/components/products/products.js

```diff
@@ -11,7 +11,7 @@ const Products = () => {
       <div class="space-2rem"></div>
       <div class="vertical-card-outer-container">
         {products.map((product) => {
-          return <ProductCard key={product.id} value={product} />;
+          return <ProductCard key={product.id} {...product} />;
```

Comment:
I can't understand what is happening here

Reply:
we can pass props like this too !! Just a another way .